packages/block-library/src/gallery/edit.js

```diff
@@ -313,65 +308,32 @@ class GalleryEdit extends Component {
 						/> }
 						<ToggleControl
 							label={ __( 'Crop Images' ) }
+							separatorType="fullWidth"
```

Comment:
Unfortunately, the separatorType property on the web is added to the dom and it causes a react warning "Warning: React does not recognize the `separatorType` prop on a DOM element.". I guess this property should be added only on native using a platform check (defined at the top).

Reply:
Nice suggestion! I've extracted this as a static const, so the prop should no longer appear for web. I've also added an explanatory comment, noting that we can remove this when the defaults are consistent on mobile controls.